lambdaclass/champions_of_mirra
# Add new attribute name to ConfigMap in messages.proto

> [!warning]
> Merge alongside [this server PR](https://github.com/lambdaclass/mirra_backend/pull/808)

## Motivation

Catch map's name received from the BackEnd

## Summary of changes

[Add map's name to messages.proto](https://github.com/lambdaclass/champions_of_mirra/pull/1996/commits/1ef54752fbf78a2142d2b63e3450ff177d0411c2)

## How has this been tested?

 - Add this line of code `Debug.Log(GameServerConnectionManager.Instance.config.Map);` in `client/Assets/Scripts/CustomLevelManager.cs:50`
 - Start a game a you'll see in Unity's shell the map's name 
 
## Checklist
- [x] I have tested the whole game after applying the changes, not only the affected areas.
